src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const ths = document.querySelectorAll('th');
+const table = document.querySelector('table');
+const tbody = document.querySelector('tbody');
+
+let sort = false;
+
+ths.forEach(th => th.addEventListener('click', event => {
+  const cellsTable = [...table.rows[0].cells];
+  let rowsAr = [];
+
+  for (let i = 0; i < cellsTable.length; i++) {
+    if (cellsTable[i] === event.currentTarget) {
+      if (sort) {
+        rowsAr = ([...tbody.rows].sort((rowA, rowB) => {
+          if (event.currentTarget === cellsTable[3]) {
```

Comment:
```suggestion
          if (event.currentTarget === cellsTable[3]) {
```
а що таке cellsTable[3]? Якщо завтра колонку перемістять в інше місце все поламається. Пропоную відштовхуватися від типу контенту всередині. Думай ще